init/kubic-init.systemd.conf

```
@@ -3,6 +3,9 @@
 [Unit]
 Description=Kubic Initialization Container
 After=crio.service docker.service
+# Check if configuration files exist
+ConditionPathExists=/etc/kubernetes/admin.conf
```

This contdition should be negated, as the `kubic-init` should not be started if `/etc/kubernetes/admin.conf` exists.
```suggestion
ConditionPathExists=!/etc/kubernetes/admin.conf
```